src/primal_constraint_coefficients.jl

```diff
@@ -0,0 +1,134 @@
+function fillAi(Ai::Vector{T}, saf::SAF{T}) where T
```

Comment:
we should probably hold ScalarAffineTerms in ScalarAffineFunctions inestead of using vectors, but that can be for the future.